cypress/utils/utils.ts

```diff
@@ -1331,3 +1331,15 @@ export function doesExistText(str: string, toBePresent: boolean): void {
         cy.contains(str).should("not.exist");
     }
 }
+
+export function validateShortInput(selector, anotherSelector?: string): void {
+    inputText(selector, "ab");
+    if (anotherSelector) click(anotherSelector);
```

Comment:
what is anotherSelector ?

Reply:
After entering 2 symbols I need to click to another field to enable validation in original field. So that is selector of that second field where I want to click. In case that name is too long for example - no need to click at another field, that is why is parameter is optional.

Reply:
Ok Got it .